x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -341,6 +384,19 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
     [isAlertTableLoading, loading]
   );
 
+  const dataViewSpec = useMemo(
```

Comment:
Again it should exist in a wrapper created for `Alert Page Filters`. I will see if I have time to create and push it along.

Reply:
Same as https://github.com/elastic/kibana/pull/179243#discussion_r1562682027, moved to [detection_engine_filters.tsx](https://github.com/umbopepato/kibana/blob/3c3e8e13fc3b79e7cbf7ebad5a425e1a42581be2/x-pack/plugins/security_solution/public/detections/components/detection_engine_filters/detection_engine_filters.tsx)

Reply:
Thanks.